### DISCUSSION TITLE
single connection mode fails

### DISCUSSION THREAD

**Discussion by MarusDod** [enisdenjo/graphql-sse#115] [General]
the current implementation will fail in single connection mode when running under a load balancer with multiple processes/containers since there is no guarantee that the second request will hit the same server instance as the one with the stream token. 

ideally there should be a strategy for storing tokens in shared memory like redis

**Comment by enisdenjo** [comment]
Not under the scope of graphql-sse at the moment. I assume a [custom implementation of the handler](https://github.com/enisdenjo/graphql-sse/blob/master/src/handler.ts) can take place handling a distributed services setup.
